### DISCUSSION TITLE
Multitasking with multiple monitors

### DISCUSSION THREAD

**Discussion by carniz** [elementary/gala#1073] [Ideas]
(Sorry if there's an existing discussion for this topic somewhere else but since I couldn't find anything with a quick search I started a new one here)

I have an external monitor connected to my laptop and I've been playing with the idea of how multitasking should work when more than one display is present, and here's a mockup of how I imagine it could work:

![Multitasking-mockup](https://user-images.githubusercontent.com/1648530/106386736-c1c50d80-63d6-11eb-950d-391ee32995b6.png)

Since the current row of icons reflects what's on the primary display, I simply drew the conclusion "one row == one display" and added a second row for the second display. With this approach one could move applications between displays by dragging their icons between the rows, and cycle between displays with the arrow up&down keys. The left&right arrow keys would move between the workspaces on each display.

Thoughts/comments?

**Comment by fd6130** [comment]
Nice idea, surely it will bring huge convenient for users who use multiple monitor.
